autospec/pkg_integrity.py

```diff
@@ -231,15 +231,10 @@ def get_signature_url(package_url):
         return package_url + '.sig'
     else:
         try:
-            st = head_request(package_url + '.asc')
-            if st == 200 or st == 302:
-                return package_url + '.asc'
-            st = head_request(package_url + '.sig')
-            if st == 200 or st == 302:
-                return package_url + '.sig'
-            st = head_request(package_url + '.sign')
-            if st == 200 or st == 302:
-                return package_url + '.sign'
+            iter = (package_url + "." + ext for ext in ("sig", "asc", "sign"))
```

Comment:
So I'm just noticing this now but the order changed to prefer asc over sig (I'm unsure how important that is mind you, maybe @fenrus75 can comment on that) and this commit is changing it back.

Reply:
there's a unit test that would fail if `.sig` is not preferred.

Reply:
ok, so it has been changed with the recent merge, so the tests do not care what's the preferences. i'll just make it the same as the current master.